src/ParseObject.js

```diff
@@ -1251,6 +1252,9 @@ class ParseObject {
     if (options.hasOwnProperty('installationId') && typeof options.installationId === 'string') {
       saveOptions.installationId = options.installationId;
     }
+    if (options.hasOwnProperty('context') && typeof options.context === 'object') {
+      saveOptions.context = options.context;
```

Comment:
Can you add documentation above for options?

Reply:
Ah right, got that as TODO above.

Reply:
done